data/src/buffer.rs

```diff
@@ -247,16 +247,58 @@ impl Default for Timestamp {
     }
 }
 
+#[derive(Debug, Clone)]
+pub enum BacklogText {
+    Text(String),
+    Hidden,
+}
+
+impl<'de> Deserialize<'de> for BacklogText {
+    fn deserialize<D>(deserializer: D) -> Result<Self, D::Error>
+    where
+        D: serde::Deserializer<'de>,
+    {
+        use serde::de;
+        #[derive(Deserialize)]
+        #[serde(untagged)]
+        enum T {
+            String(String),
+            Bool(bool),
+        }
+
+        match T::deserialize(deserializer)? {
+            T::String(conf) => Ok(Self::Text(conf)),
+            T::Bool(val) => {
+                if val == false {
+                    Ok(Self::Hidden)
+                } else {
+                    Err(de::Error::custom(
+                        "true is not a valid value for buffer.backlog_separator.text",
```

Comment:
I feel like we could support true and turn it into default value just to support both boolean values. What do you think? Feels better to always support both false AND true.